Src/Sarif.PatternMatcher/ValidatorsCache.cs

```diff
@@ -417,6 +417,8 @@ internal static FlexMatch ConstructRegionFromGroups(IDictionary<string, FlexMatc
                             isValidDynamic = null;
                         }
 
+                        MethodInfo clear = type.GetMethod("Clear");
```

Comment:
>GetMethod [](http://example.com/codeflow?start=48&length=9)

Looks like you didn't complete the end to end of this change? For this sort of thing, an API change, I'd suggest doing that. What I'd do is plumb the complete API change through with a single validator, then if I liked it, would apply the change mechanically to everything else.